packages/ui/feature-builder-store/src/lib/store/flow/flow.effects.ts

```diff
@@ -220,24 +221,34 @@ export class FlowsEffects {
         const genSavedId = UUID.UUID();
         let flowOperation: FlowOperationRequest;
         switch (action.type) {
-          case FlowsActionType.UPDATE_TRIGGER:
+          case FlowsActionType.UPDATE_TRIGGER: {
+            const op =
+              FlowStructureUtil.removeAnySubequentStepsFromFlowOperationStep(
+                action.operation
+              ) as typeof action.operation;
```

Comment:
Please don't case, and removeAnySubequentStepsFromFlowOperationStep make return type update operation and map values there.